src/main/java/seedu/task/model/task/Task.java

```diff
@@ -15,29 +15,30 @@
     private Name name;
     //private DateTime openTime;
     //private DateTime closeTime;
-    //private boolean isImportant;
+    private boolean isCompleted;
 
     private UniqueTagList tags;
 
     /**
      * Every field must be present and not null.
      */
-    public Task(Name name, UniqueTagList tags) {
+    public Task(Name name, UniqueTagList tags,boolean isCompleted) {
         // open time, urgent, and close time can be null
         assert !CollectionUtil.isAnyNull(name, tags);
         this.name = name;
         //TODO: set default values
         //this.openTime = openTime;
         //this.closeTime = closeTime;
         //this.isImportant = isImportant;
-        this.tags = new UniqueTagList(tags); // protect internal tags from changes in the arg list
+        this.tags = new UniqueTagList(tags);
+        this.isCompleted=false;  // protect internal tags from changes in the arg list
```

Comment:
`isCompleted` is passed into the constructor. Why false?

Also, that comment is for the previous line.